code/game/machinery/doors/windowdoor.dm

```diff
@@ -114,7 +114,7 @@
 	if(!open())
 		return
 	autoclose = TRUE
-	if(check_access(null))
+	if(check_access(null) && !associated_lift)
```

Comment:
I'd rather this be done by overriding check_access, rather then adding a secondary check here.
I also don't really like the associated_lift var on the windoor type, smells wrong

Reply:
Done, removed it from all windoors. Now it's only applied to the tram subtype.